cluster/cluster.go

```diff
@@ -522,8 +531,12 @@ SELECT
 	s.dbid,
 	s.content as contentid,
 	s.role,
+	s.preferred_role,
```

Comment:
I don't think this works.

```suggestion
	s.preferred_role as preferredrole,
```

We should also update `GetSegmentConfiguration` test to reflect this change.